### DISCUSSION TITLE
Schmid factor issue

### DISCUSSION THREAD

**Discussion by King-wang-H** [mtex-toolbox/mtex#1128] [Ask Anything]
Hi MTEX Users,
I have recently attempted to calculate the Schmid factor for tensile twins in magnesium alloys using MTEX.Now, I would like to ask  a question. Can the Schmidfactor command be applied to complex stress states (triaxial force, matrix type)?Or is it only applicable to Schmid Factor calculation under uniaxial stress?
This is the code that I used,
[
![1 3 I9WZJV39RUZRF K_6$T](https://user-images.githubusercontent.com/85743677/125287516-96642880-e34f-11eb-9702-2b09fec220a6.png)
](url)
I hope someone can help me solve the puzzle, thank you!

**Comment by kilir** [comment]
Hi,

you should be able to use any stress tensor you want.

Cheers,
Rüdiger
